### DISCUSSION TITLE
Integrate with Echo Framework

### DISCUSSION THREAD

**Discussion by raymondlukanta** [zitadel/oidc#256] [Q&A]
I have an existing code repository using Go Echo Framework (https://echo.labstack.com/).
Is there any tutorial/ guide on how to integrate this oidc library with echo framework?

Thanks

**Comment by adlerhurst** [comment]
I think echo provides a wrapper to handle `http.Handler` and `http.HandlerFunc`: https://github.com/labstack/echo/issues/1781
